### PR TITLE
[DataGrid] Improve ptBR locale

### PR DESCRIPTION
**Pull request by aline-matos**
Add missing export translations.

### CODE REVIEW

**Comment by oliviertassinari** [comment]
How about we use this opportunity to add the same keys commented in the other locales?

https://github.com/mui-org/material-ui-x/blob/f5221ef2a1b4345c244cfc3704de67b39472e54a/packages/grid/_modules_/grid/constants/localeTextConstants.ts#L28-L31

It will make it easier for the other translators to handle.

File: packages/grid/_modules_/grid/locales/ptBR.ts [hunk 1]
```diff
@@ -26,6 +26,11 @@ const ptBRGrid: Partial<GridLocaleText> = {
   toolbarFiltersTooltipShow: 'Exibir filtros',
   toolbarFiltersTooltipActive: (count) =>
     `${count} ${count !== 1 ? 'filtros' : 'filtro'} ${count !== 1 ? 'ativos' : 'ativo'}`,
+  
+  // Export selector toolbar button text
+  toolbarExport: 'Exportar',
+  toolbarExportLabel: 'Exportar',
+  toolbarExportCSV: 'Download como CSV',
```
Comment (m4theushw) [inline packages/grid/_modules_/grid/locales/ptBR.ts:33]: It's better to translate the entire phrase.

```suggestion
  toolbarExportCSV: 'Baixar como CSV',
```

**Comment by m4theushw** [comment]
@oliviertassinari Side note: the `toolbarExportLabel` key is not used. We could remove it from the other locales in another PR.

**Review by m4theushw — APPROVED** [review]

**Comment by oliviertassinari** [comment]
> Side note: the toolbarExportLabel key is not used. We could remove it from the other locales in another PR.

@m4theushw Great 👁️ for the details! It was added in #1030, maybe it was refactored and forgot along the way. In this case, no need to translate it.

**Comment by oliviertassinari** [comment]
> maybe it was refactored and forgot along the way. In this case, no need to translate it.

Actually, it depends, It looks like @DanailH added it so that a button can have a label if only the icon is visible. 

https://github.com/mui-org/material-ui-x/blob/45093913e390b9c71b89a3debf76a58a46e68a2f/packages/grid/_modules_/grid/components/toolbar/GridToolbarDensitySelector.tsx#L99

👍 for a new issue/a follow-up to make this consistent. I would personally argue that for the only icon use case, https://next.material-ui.com/system/screen-readers/#visually-hidden-elements has the potential to be more elegant. Less code, fewer translations.

**Comment by oliviertassinari** [comment]
> maybe it was refactored and forgot along the way. In this case, no need to translate it.

@m4theushw Actually, it depends, It looks like @DanailH added it so that a button can have a label if **only the icon is visible**. 

https://github.com/mui-org/material-ui-x/blob/45093913e390b9c71b89a3debf76a58a46e68a2f/packages/grid/_modules_/grid/components/toolbar/GridToolbarDensitySelector.tsx#L99

👍 for a new issue/a follow-up to make this consistent between the different toolbar buttons.

I would personally argue that for the only icon use case, https://next.material-ui.com/system/screen-readers/#visually-hidden-elements has the potential to be more elegant. Less code, fewer translations. But If we want to make it work in a responsive way (hide the label on small screens), it might not fly.

**Review by dtassone — APPROVED** [review]

**Pull request merged**